### PR TITLE
Add documentation of the Openplantbook integration

### PR DESCRIPTION
**Pull request by Olen**
## Proposed change

Documentation for the Openplantbook integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

Brands are coming as soon as the owner of the upstream API provides them.

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/42359
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1993
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/openplantbook.markdown [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+{% endconfiguration %}
+
+
+## Examples
+
+Two service calls are added by this integration:
+
+`openplantbook.search` searches the API for plants matching a string. The search result is added to the entity `openplantbook.search_result` with the number of returned results as the `state` and a list of results in the state attributes.
+
+{% raw %}
```
Comment (klaasnicolaas) [inline source/_integrations/openplantbook.markdown]: You can remove the `raw` and `endraw` tags for this piece of yaml 😉

File: source/_integrations/openplantbook.markdown [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  * capsicum annuum -> Capsicum annuum
+  * capsicum baccatum -> Capsicum baccatum
+  * capsicum bomba yellow red -> Capsicum Bomba yellow red
+  * capsicum chinense -> Capsicum chinense
+(...)
+
+
+`openplantbook.get` gets detailed data for a single plant. It takes the `pid` from `openplantbook.search_result` as a parameter and the result is added to the entity `openplantbook.<species name>` with the different max/min values etc. set as attributes.
+
+{% raw %}
```
Comment (klaasnicolaas) [inline source/_integrations/openplantbook.markdown]: Here also

File: source/_integrations/openplantbook.markdown [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+* Max moisture: 65
+* Min moisture: 20
+* Max temperature: 35
+* Image: https://.../capsicum%20annuum.jpg
+
+## Caching
+All plant data is cached as it is not expected to change much.  However fresh plant data is fetched if the `get` service is run on a species where the previous get was done more than 24 hours ago. If you have done many requests, and have a lot of entries under the `openplantbook.<species>` hierarchy, you can clean up the cache by issuing a `openplantbook.clean_cache` service.  The service takes one paramater - the age of the data to keep cached (defaults to 24).
+
+
+{% raw %}
```
Comment (klaasnicolaas) [inline source/_integrations/openplantbook.markdown]: And here also 😄

File: source/_integrations/openplantbook.markdown [hunk 4]
```diff
@@ -0,0 +1,117 @@
+---
+title: OpenPlantbook
+description: Instructions on how to setup OpenPlantbook with Home Assistant.
+ha_category:
+  - Environment
+ha_iot_class: Cloud Polling
+ha_release: 0.116
+ha_quality_scale: internal
```
Comment (klaasnicolaas) [inline source/_integrations/openplantbook.markdown]: internal isn't the right option in my opinion, see: https://www.home-assistant.io/docs/quality_scale/

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/openplantbook.markdown (hunk 4 above)
Comment (Olen) [reply source/_integrations/openplantbook.markdown]: Sorry.  Missed that when copying from another integration

File: source/_integrations/openplantbook.markdown (hunk 1 above)
Comment (Olen) [reply source/_integrations/openplantbook.markdown]: Removed raw-tags for yaml. But kept them for jinja2

File: source/_integrations/openplantbook.markdown [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+ha_codeowners:
+  - '@Olen'
+ha_domain: openplantbook
+---
+
+The `openplantbook` component allows you to search and get data from the [OpenPlantbook API](https://open.plantbook.io/) 
+
+## Configuration
+
+The integration is set up using the GUI.  You must have a valid `client_id` and `secret` from OpenPlantbook to set up the integration.
```
Comment (klaasnicolaas) [inline source/_integrations/openplantbook.markdown]: Here you could put some additional information about how to set up the integration in the GUI, there are now plenty of examples to be found with other integrations 😉

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/openplantbook.markdown (hunk 5 above)
Comment (Olen) [reply source/_integrations/openplantbook.markdown]: Thanks for helping out.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**